lib/yunohost/backup.py

```diff
@@ -169,8 +171,22 @@ def _clean_tmp_dir(retcode=0):
                 msignals.display(m18n.n('unbackup_app', app_id),
                                  'warning')
                 continue
+                
+            # Copy the app restore script
+            app_restore_script = app_setting_path + '/scripts/restore'
+            if os.path.isfile(app_script):
+                try:
+                    filesystem.mkdir(tmp_app_dir, 0750, True, uid='admin')
+                    shutil.copy(app_restore_script, tmp_app_dir)
```

Comment:
This is not useful, since the whole folder `/etc/yunohost/apps/<app_id>` is already copied.